src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate.cs

```diff
@@ -458,6 +479,41 @@ public virtual string ToString(bool fVerbose)
             return sb.ToString();
         }
 
+        public virtual void Import(byte[] rawData)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Import(byte[] rawData, string password, X509KeyStorageFlags keyStorageFlags)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Import(byte[] rawData, SecureString password, X509KeyStorageFlags keyStorageFlags)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Import(string fileName)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Import(string fileName, string password, X509KeyStorageFlags keyStorageFlags)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Import(string fileName, SecureString password, X509KeyStorageFlags keyStorageFlags)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public virtual void Reset()
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
I can change it however the corefx implementation did not clear out the private fields (only Pal object). I changed it to also clear out the fields -- which doesn't change semantics (we always throw if accessing a member after Dispose) but it will help with future Import support and helps out the garbage collector a bit (avoiding moving those objects to gen1 if there are no other refs)